src/MongoDB.Driver.Core/Core/Servers/ServerMonitor.cs

```diff
@@ -29,8 +29,27 @@ namespace MongoDB.Driver.Core.Servers
 {
     internal sealed class ServerMonitor : IServerMonitor
     {
+        #region static
         private static readonly TimeSpan __minHeartbeatInterval = TimeSpan.FromMilliseconds(500);
 
+        private static bool ShouldSdamDescriptionChangedEventsBePublished(ServerDescription oldDescription, ServerDescription newDescription)
+        {
+            var canBeConsideredAsEqual =
+                EndPointHelper.Equals(oldDescription.EndPoint, newDescription.EndPoint) &&
+                oldDescription.Type == newDescription.Type &&
+                object.Equals(oldDescription.WireVersionRange, newDescription.WireVersionRange) &&
```

Comment:
Nit 1: I believe the `object` qualifiers in this function are redundant
Nit 2: Would it be clearer to say `WireVersionRange.Equals`, `TagSet.Equals`, and `ElectionId.Equals`?
Question: What does `// me` stand for?

Reply:
>Nit 1: I believe the object qualifiers in this function are redundant

yes, but we do it in this way in a lot of other places. So I just repeated it.

Reply:
>Nit 2: Would it be clearer to say WireVersionRange.Equals, TagSet.Equals, and ElectionId.Equals?

it would not be the same logic. `object.Equals` protects variables from `NullReferenceException`.

>Question: What does // me stand for?

I guess it was discussed here: https://mongodb.slack.com/archives/GQ27C9N7J/p1575490389037500
Let me know if you still have any questions.

Reply:
Thanks! And oops you're right about the object.Equals providing protection from nulls, reference, thanks for the explanation! :)